packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -91,20 +94,30 @@ def config_all_loggers(
     *,
     log_format_local_dev_enabled: bool,
     logger_filter_mapping: dict[LoggerName, list[MessageSubstring]],
+    tracing_settings: TracingSettings | None,
 ) -> None:
     """
     Applies common configuration to ALL registered loggers
     """
-    fmt = DEFAULT_FORMATTING
     the_manager: logging.Manager = logging.Logger.manager
     root_logger = logging.getLogger()
 
     loggers = [root_logger] + [
         logging.getLogger(name) for name in the_manager.loggerDict
     ]
 
+    fmt = DEFAULT_FORMATTING
+    if tracing_settings is not None:
+        fmt = (
+            "log_level=%(levelname)s | log_timestamp=%(asctime)s | log_source=%(name)s:%(funcName)s(%(lineno)d) | log_uid=%(log_uid)s "
```

Comment:
Try formatting them  as
```
DEFAULT_FORMATTING = (
    "log_level=%(levelname)s "
    "| log_timestamp=%(asctime)s "
    "| log_source=%(name)s:%(funcName)s(%(lineno)d) "
    "| log_uid=%(log_uid)s "
    "| log_oec=%(log_oec)s"
    "| log_msg=%(message)s"
```


BTW, do not forget i also added `log_oec`

Reply:
Good point. Done